recipes/pyfizi/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyfizi" %}
+{% set version = "0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a4b47cf057fce63e6d912f87d0e497065436f4b501698543863b5c82970c56f6
+
+build:
+  skip: True  # [win or py!=36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.14.5
```

Comment:
I think only `python` and `pip` are needed in the `host` section. (For a Python package, this only needs to contain dependencies which are actually used by `setup.py`.)

Reply:
Okay I'll retry with those removed. I had generated the initial script with `conda skeleton pyfizi` and edited from there.